extensions/vault/runtime/src/main/java/io/quarkus/vault/runtime/client/MutinyVertxClientFactory.java

```diff
@@ -19,8 +19,8 @@
     public static WebClient createHttpClient(Vertx vertx, VaultBootstrapConfig vaultBootstrapConfig, TlsConfig tlsConfig) {
 
         WebClientOptions options = new WebClientOptions()
-                .setConnectTimeout((int) vaultBootstrapConfig.connectTimeout.getSeconds())
-                .setIdleTimeout((int) vaultBootstrapConfig.readTimeout.getSeconds());
+                .setConnectTimeout((int) vaultBootstrapConfig.connectTimeout.toMillis())
```

Comment:
The Javadoc of `ClientOptionsBase` says that the timeout is in milliseconds

Reply:
good catch for the connect timeout.
however, the idle timeout is in seconds:
```
Set the idle timeout, default time unit is seconds. Zero means don't timeout. This determines if a connection will timeout and be closed if no data is received within the timeout. If you want change default time unit, use setIdleTimeoutUnit(TimeUnit)
Params:
idleTimeout – the timeout, in seconds
Returns:
a reference to this, so the API can be used fluently
```

Reply:
That sounds about right :]

Reply:
Wow, that is not intuitive...

Reply:
Fixed